### ISSUE TITLE
kubernetes-gce-serial failing consistently since 4pm yesterday: high mem usage for runtime

### ISSUE THREAD

**Issue by mikedanese** [kubernetes/kubernetes#31281]
```
	
[k8s.io] Kubelet [Serial] [Slow] [k8s.io] regular resource usage tracking resource tracking for 100 pods per node

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/kubelet_perf.go:277 Aug 23 07:33:58.540: Memory usage exceeding limits: node jenkins-e2e-minion-group-y5s8: container "runtime": expected RSS memory (MB) < 314572800; got 539475968 node jenkins-e2e-minion-group-hi68: container "runtime": expected RSS memory (MB) < 314572800; got 543174656 node jenkins-e2e-minion-group-rmg4: container "runtime": expected RSS memory (MB) < 314572800; got 529215488 /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/kubelet_perf.go:153


[k8s.io] Kubelet [Serial] [Slow] [k8s.io] regular resource usage tracking resource tracking for 0 pods per node

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/kubelet_perf.go:277 Aug 23 07:57:19.675: Memory usage exceeding limits: node jenkins-e2e-minion-group-y5s8: container "runtime": expected RSS memory (MB) < 89128960; got 110391296 node jenkins-e2e-minion-group-hi68: container "runtime": expected RSS memory (MB) < 89128960; got 105758720 /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/kubelet_perf.go:153


[k8s.io] Kubelet [Serial] [Slow] [k8s.io] regular resource usage tracking resource tracking for 35 pods per node

/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/kubelet_perf.go:277 Aug 23 08:20:18.752: Memory usage exceeding limits: node jenkins-e2e-minion-group-y5s8: container "runtime": expected RSS memory (MB) < 157286400; got 257867776 node jenkins-e2e-minion-group-hi68: container "runtime": expected RSS memory (MB) < 157286400; got 251043840 node jenkins-e2e-minion-group-rmg4: container "runtime": expected RSS memory (MB) < 157286400; got 254713856 /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/kubelet_perf.go:153
```

**Comment by yujuhong** [comment]
The test started failing in build 1994 in kubernetes-e2e-gce-serial, but there is no obvious suspect in that batch or PRs. Still digging.

**Comment by yujuhong** [comment]
From build 1994:
```
16:06:45 KUBE_OS_DISTRIBUTION=gci
```

@vishh did we change kubernetes-gce-serial suite to run GCI on all the nodes?

**Comment by yujuhong** [comment]
Going to fix be fixed by https://github.com/kubernetes/test-infra/pull/438

**Comment by vishh** [comment]
Fixed via kubernetes/test-infra#438
